tests/ops/models/test_base.py

```diff
@@ -75,28 +75,37 @@ def test_update_key(db: Session, storage_config, privacy_request):
     assert "complete" == privacy_request.status.value
 
 
-def test_save_key(db: Session, storage_config, privacy_request):
-    # Test prevent saving bad keys.
-    storage_config.key = "bad key"
-    with pytest.raises(KeyValidationError) as exc:
-        storage_config.save(db)
-
-    assert str(exc.value) == "Key 'bad key' on StorageConfig is invalid."
-
-    # Test key required on applicable models
-    storage_config.key = None
-    with pytest.raises(KeyValidationError) as exc:
+class TestSaveKey:
+    def test_save_key_invalid(self, db: Session, storage_config):
+        "Test prevent saving bad keys."
+        storage_config.key = "bad key"
+        with pytest.raises(FidesValidationError) as exc:
+            storage_config.save(db)
+
+        error_output = str(exc.value)
+        expected_output = "FidesKeys must only contain alphanumeric characters, '.', '_', '<', '>' or '-'. Value provided: bad key"
+        print(error_output)
```

Comment:
I'd remove these extra print statements out of these tests -

Reply:
Why? I find them extremely useful when debugging tests, and they only show up in the `stdout` if the test fails, so I consider them not only harmless but actually free debugging help. Pytest captures `print` statements on a per-test basis